src/utils/config.ts

```diff
@@ -329,11 +329,14 @@ export const RECAPTCHA_SCORE_THRESHOLD = 0.5;
 export const GET_MOST_LIKED_ITEMS_MAXIMUM = 50;
 export const GET_MOST_RECENT_ITEMS_MAXIMUM = 50;
 
+// user feedback reporting
+export const USER_FEEDBACK_PREFIX = '/user-feedback';
```

Comment:
I think it would make more ssens to define this in `src/services/userFeedback/index.ts` directly. Like it is done for the `/members` route.